### PR TITLE
front: i18n for editor's forms

### LINKED ISSUE #2231: Translate forms in French
**Describe the current situation**
Today, in the edition module, indications on all the forms to edit the different elements are in English 

**Describe your improvement**
They should be traslated in French

### PR DESCRIPTION
**Pull request by sim51**
This PR fixes #2231 and  contains : 

- upgrade of rjsf to use the version 5
- using i18n validator for generic validator (like mandatory)
- convert the jsonschema to an i18n json schema
-  in dev mode, missing keys are written in console debug

/!\ Locale files were not populated, just a few  example are done

### CODE REVIEW

File: front/public/locales/fr/translation.json
Comment (nicolaswurtz) [inline front/public/locales/fr/translation.json:1]: Please create different files dedicated to translation for Editor, thanx !

**Review by nicolaswurtz — CHANGES_REQUESTED** [review]
As said before, it's time ! Please create different files dedicated to translation for Editor, thanx :)

File: front/public/locales/fr/translation.json
Comment (sim51) [reply front/public/locales/fr/translation.json:1]: doing it right now

Comment (sim51) [reply front/public/locales/fr/translation.json:1]: Done in [c4a81da](https://github.com/DGEXSolutions/osrd/pull/3845/commits/c4a81da1370ba2415f5c6fb92a4d6cdc8693c9df)

**Comment by alexandredamiron** [comment]
Plz rebase and check yarn.lock. @nicolaswurtz plz rereview

**Pull request closed without merging**